src/datasources/perf-ds/partials/query.editor.html

```diff
@@ -87,6 +87,21 @@
                     >
                 </div>
             </div>
+            <div class="gf-form-inline">
+                <div class="gf-form max-width-25">
+                    <label class="gf-form-label query-keyword width-10">
+                        Fallback Attribute <i class="fa fa-tag"></i>
```

Comment:
It'd be nice to be able to use the  "Attribute selection modal" here to be able to lookup the value. Otherwise you need to either know the attribute name, or use the other modal, copy the value and dismiss it so that it doesn't overwrite the existing attribute value.